### ISSUE TITLE
featureCV unit test

### ISSUE THREAD

**Issue by lgatto** [lgatto/MSnbase#212]
The `featureCV` [unit test](https://github.com/lgatto/MSnbase/blob/master/tests/testthat/test_MSnSet.R#L249-L272) fails because of a syntax error 

```r
 cv <- matrix(c(0.5, 1/4.5, 1/7, 1/9.5),
                 nrow=2, ncol=2, dimnames=list(c("A", "B"), c("CV.1", "CV.2"))
```
should be (missing closing paren)

```r
 cv <- matrix(c(0.5, 1/4.5, 1/7, 1/9.5),
                 nrow=2, ncol=2, dimnames=list(c("A", "B"), c("CV.1", "CV.2")))
```
and difference in the first `expect_equal`.

This is strange, as it would have failed before. Not sure if I somehow messed up locally? @sgibb, could you have a look, in case I miss something.

**Comment by sgibb** [comment]
That's indeed strange, because I am sure it wasn't failing locally. But it is fixed now in 94d4679f52e32dcfa174a8c46b3ab7a32feae668

**Comment by lgatto** [comment]
Thank you for this!
